src/reference.js

```diff
@@ -309,12 +309,13 @@ class DocumentReference {
    * });
    */
   getCollections() {
-    let request = {
+    const request = {
       parent: this._referencePath.formattedName,
     };
+    const requestTag = Firestore.requestTag();
 
     return this._firestore
-      .request('listCollectionIds', request)
+      .request('listCollectionIds', request, requestTag)
```

Comment:
Feel like in many places like this you can just inline the call to Firestore.requestTag() and avoid having to declare an additional variable/constant.

Reply:
I think this is actually the only place. I have already done this in some places, and in the remaining callsites, `requestTag` gets referenced more than once.